### DISCUSSION TITLE
Maximizing a window. How?

### DISCUSSION THREAD

**Discussion by heysokam** [siduck/chadwm#183] [Q&A]
What's the name for maximizing a window on dwm/chadwm?
I can maximize windows that supply a maximize icon, but I cannot get them to stay maximized.

_Note: I know about he monocle layout, but I'm talking about maximizing in any situation, not inside that layout._
_Also, maximizing doesn't work for monocle windows or floating windows either, so I figure I must be missing something._

**Comment by ghost** [comment]
do you mean maximizing by making it fullscreen?

you can check this 
https://github.com/siduck/chadwm/blob/main/chadwm/config.def.h#L159
if you mean so

**Reply by heysokam** [reply]
This ignores the bar completely.
I just mean maximizing a floating or non-floating window to take the full size that is available for windows.
It works in awesomewm, but I don't know if its inherited behavior from dwm or an added awesomewm feature.

In awesomewm, for reference, this makes that single window maximized, and the rest of the windows stay in whatever tiling layout is being used.

I don't mind if the window has to be marked as "floating" for it to work... its just not maximizing any windows at all, no matter their floating state.

**Comment by heysokam** [comment]
> do you mean maximizing by making it fullscreen?

https://dwm.suckless.org/patches/maximize/
@Subhashis2007 I guess I meant this patch. I didn't know it existed.
I figure it has not been applied :shrug:

**Reply by ghost** [reply]
Looks like it....
